.github/workflows/ci.yml

```diff
@@ -376,6 +376,11 @@ jobs:
     - name: Print fancy log report
       if: ${{ always() }}
       run: python utils/scripts/markdown_logs.py >> $GITHUB_STEP_SUMMARY
+    - name: Run TELEMETRY_MESSAGE_BATCH_EVENT_ORDER scenario
```

Comment:
Can you move up this step, just below `Run SAMPLING scenario` ? otherwise, you test result and artifact will be missing